tests/load/locustfiles/locustfile.py

```diff
@@ -96,54 +134,127 @@ def connect(self) -> None:
         self.ws = create_connection(
             self.environment.parsed_options.websocket_url,
             header={"Origin": "http://localhost:1337"},
-            ssl=False,
+            timeout=30,  # timeout defaults to None
```

Comment:
I wonder if this is too high -- it might be preferable to fail faster when things aren't working as expected.

Reply:
oh! that's my bad, at one point I thought the timeout had to be greater than our wait_time interval. What's a good amount 5?

Reply:
It's very short in our integration tests (1) but 5 works